psi4/src/psi4/dfocc/ccd_tpdm.cc

```diff
@@ -39,534 +39,143 @@ namespace dfoccwave {
 void DFOCC::ccd_tpdm() {
     timer_on("tpdm");
 
- // RHF
```

Comment:
Maybe revert all or just the non-throw changes from dfocc/ccd_tpdm.cc and dfocc/tensor.cc ? dfocc still needs some reconciliation with uncommitted changes, and all the spacing changes here are going to make that hard. sorry for the trouble. otherwise, lgtm!

Reply:
Done. Yeah, stupidly I ran clang-format before committing the changes, and then realized what a mess it had made.